src/utterances.ts

```diff
@@ -25,6 +25,15 @@ function loadIssue(): Promise<Issue | null> {
   return loadIssueByTerm(page.issueTerm as string);
 }
 
+function injectCustomCss() {
+  if (page.customCss !== null) {
+    let link = document.createElement('link');
+    link.rel = 'stylesheet';
+    link.href = page.customCss;
+    document.head.appendChild(link);
+  }
+}
+
```

Comment:
```ts
function loadCustomStylesheet() {
  if (page.stylesheet) {
    const link = document.createElement('link');
    link.rel = 'stylesheet';
    link.href = page.stylesheet;
    link.setAttribute('crossorigin', 'anonymous');
    document.head.appendChild(link);
    return new Promise(resolve => { link.onload = resolve; });
  }
  return Promise.resolve();
}
```

(delay rendering until custom stylesheet is loaded to avoid flash of unstyled/incorrectly-styled content)